src/jaegertracing/Transport.h

```diff
@@ -17,6 +17,8 @@
 #ifndef JAEGERTRACING_TRANSPORT_H
 #define JAEGERTRACING_TRANSPORT_H
 
+#include "jaegertracing/Compilers.h"
```

Comment:
Should appear last, where Jaeger headers always go. I believe `.clang-format` describes the ideal ordering.